codespell_lib/data/dictionary.txt

```diff
@@ -28571,11 +28592,20 @@ trnsfered->transferred
 trnsfers->transfers
 troling->trolling
 trottle->throttle
+trottled->throttled
```

Comment:
also `trotted`

Reply:
`trotted` is a valid word and valid spelling. Verified with Cambridge, Collins, Dictionary.com, Merriam-Webster, macmillan, Oxford Learners, aspell, hunspell, ispell, etc